packages/client/test/importer.js

```diff
@@ -67,6 +68,18 @@ export const importBlob = async (content) => {
   throw new Error(`Import failed`)
 }
 
+/**
+ * @param {Uint8Array} content
+ */
+export const importCar = async (content) => {
+  const car = await CarCIDIterator.fromBytes(content)
```

Comment:
Use `CarReader` for simplicity? I was trying to figure out why you're using `CarCIDIterator` and not `CarReader`? I don't think there's a reason, right?

Reply:
`CarReader` reads the whole CAR into mem. I just need `getRoots`, so this does less work, but agree, it's distracting from the purpose of this mock.